cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiff.java

```diff
@@ -171,18 +182,19 @@ private Collection<AlterConfigOp> diff(int brokerId, String desired,
                     .findFirst();
 
             String op = d.get("op").asText();
+            boolean nodeIsNotController = !nodeRef.controller();
```

Comment:
sorry to be pedantic. I always find double "negative" hard to understand. You are negating `controller` and then calling the variable with a `Not`. Tbh I would have `boolean isController = nodeRef.controller()` and then using the appropriate logic in the method where it's passed.

Reply:
That's what we had originally but changed based on the previous comment from Jakub. We could change it back to combined, but more explicit like either `isNodeBrokerAndController` or `isCombinedNode`. Wdyt?

Reply:
My first comment was suggesting to use `isCombinedMode` which was better than just `isCombined` imho. So I would go with it.

Reply:
I think, `isController` was actually clearer than `isCombined` or `isCombinedNode`. `KafkaBrokerConfigurationDiff` class is only called for brokers which is obvious from the name and `isController` returning true suggests that this broker is also a controller, that's all we care about when ignoring properties.

Reply:
I think you are right about not using double negatives. I will change this back.

Reply:
Assuming that people will know how it is or possibly even how it was used in the past is wrong. It should be named clearly based on what it is. If it is called only for combined nodes, please call it accordingly. If it is called for all controllers - call it controller.